payments/tests/conftest.py

```diff
@@ -104,6 +105,15 @@ def _generate_order(order_type: str = None):
             product=None,
             lease=None,
         )
+    elif order_type == "additional_product_order":
+        order = OrderFactory(
+            order_type=OrderType.ADDITIONAL_PRODUCT_ORDER,
+            customer=customer_profile,
+            price=random_price(),
+            tax_percentage=random_tax(),
+            product=None,
+            lease=None,
+        )
```

Comment:
To ensure that the order will have the additional product.
```suggestion
        order = OrderFactory(
            order_type=OrderType.ADDITIONAL_PRODUCT_ORDER,
            customer=customer_profile,
            price=random_price(),
            tax_percentage=random_tax(),
            product=None,
            lease=None,
        )
        OrderLineFactory(
             order=order
         )             
```

Reply:
Tried adding it but it opens kinda worm box with the model validations (starting from a lease and a customer) so IMO not worth doing.